File: service/src/main/java/com/loudsight/utilities/service/dispatcher/bridge/BridgeMessageType.java
package com.loudsight.utilities.service.dispatcher.bridge;

import com.loudsight.meta.annotation.Introspect;

@Introspect(clazz = BridgeMessageType.class)
public enum BridgeMessageType {
    DIRECT,
    DIRECT_ASYNC,
    SUBSCRIPTION,
    PUBLICATION,
    REQUEST,
    RESPONSE,
    ERROR
}

File: service/src/main/java/com/loudsight/utilities/service/dispatcher/bridge/BridgeMessageType.kt
package com.loudsight.utilities.service.dispatcher.bridge

import com.loudsight.meta.annotation.Introspect

@Introspect(BridgeMessageType::class)
enum class BridgeMessageType {
    DIRECT,
    DIRECT_ASYNC,
    SUBSCRIPTION,
    PUBLICATION,
    REQUEST,
    RESPONSE,
    ERROR
}